tests/test_mediawiki.py

```diff
@@ -212,13 +213,24 @@ class TestMediaWikiBackend_1_23(TestMediaWikiBackend):
     """MediaWiki backend tests for MediaWiki 1.23 version"""
 
     def test_fetch(self):
-        self._test_fetch_version("1.23")
-        self._test_fetch_version("1.23", reviews_api=True)
+        with self.assertLogs(logger, level='WARNING') as cm:
```

Comment:
The usage of `cm` is not clear. What's the meaning of cm? Context Manager? And why do you have to access to the message in this way: `cm[-1][2]`?

Reply:
cm (which stands for context manager) is a common variable used in combination with `assertLogs` (https://docs.python.org/3.4/library/unittest.html#unittest.TestCase.assertLogs). 
The log messages can be accessed in different ways, `cm[-1][-2]` is one of the possible ones (which is used also in other tests in Perceval). I use the index `[-2]` to access a specific message in the logs.